src/lib/core/ripple/ripple.spec.ts

```diff
@@ -86,112 +65,86 @@ describe('MdRipple', () => {
     document.body.style.margin = originalBodyMargin;
   });
 
+  function dispatchMouseEvent(type: string, offsetX = 0, offsetY = 0) {
```

Comment:
I swear this function is in like half of our test files, we should create a test utilities file sometime (not part of this PR though)